### PR TITLE
Make the homepage mobile-friendly

### PR DESCRIPTION
**Pull request by SongYuQiu**
Tasks in https://github.com/apache/incubator-kvrocks/issues/1257 are completed as follows:

- [x]  Modify the colors of Header and Footer, and update the banner.
- [ ]  Refactor introduction of the Redis Compatible feature.
  - [ ]  Include a text editor with demo.
  - [ ]  Include a pie chart displaying the proportion of implemented Redis commands.
- [x]  Add icons and descriptions for four features.
- [ ]  Add a list of logos.
- [x]  Make the homepage mobile-friendly (optional).
- [ ]  Add some animations (optional).

The icons of Replication and Cluster have been exchanged.
The second subtask is currently only occupying the page with the corresponding elements in the UI design. They will be changed to Monaco and ECharts in the next pr.
The following are screenshots of iPhone SE and iPad mini:
截屏2023-04-04 16 30 33
截屏2023-04-04 16 30 58
截屏2023-04-04 16 31 21
截屏2023-04-04 16 31 35
截屏2023-04-04 16 31 56
截屏2023-04-04 16 32 15
截屏2023-04-04 16 32 28

截屏2023-04-04 16 33 07
截屏2023-04-04 16 33 23
截屏2023-04-04 16 33 34
截屏2023-04-04 16 33 43

### CODE REVIEW

**Comment by PragmaTwice** [comment]
Hi, you can try to resolve conflicts (via merge or rebase) before we merge this PR.

**Comment by SongYuQiu** [comment]
> Hi, you can try to resolve conflicts (via git merge or rebase) before we merge this PR.

conflicts have resolved.

File: src/components/HomepageFeatures/index.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import clsx from 'clsx';
 import styles from './styles.module.css';
-import feature0 from '/img/feature-0.png';
-import feature1 from '/img/feature-1.png';
-import feature2 from '/img/feature-2.png';
-import feature3 from '/img/feature-3.png';
-import feature4 from '/img/feature-4.png';
+import feature0 from '../../../static/img/feature-0.png';
```
Comment (PragmaTwice) [inline src/components/HomepageFeatures/index.tsx:4]: Seems we can have a more concrete name for `feature0` like other images.

File: src/components/HomepageFeatures/index.tsx [hunk 2]
```diff
@@ -78,16 +77,16 @@ export default function HomepageFeatures(): JSX.Element {
               <div className={styles.blueLine}>
                 <div></div>
               </div>
-              <p>Users can access Apache Kvrocks via any Redis client</p>
-              {/* <div className={styles.redisCompatible}>
+              <p>User can use a Redis client to visit Apache Kvrocks.</p>
```
Comment (PragmaTwice) [inline src/components/HomepageFeatures/index.tsx:80]: I think you need to revert this change, since the previous words is more suitable.

Comment (SongYuQiu) [reply src/components/HomepageFeatures/index.tsx:80]: Ok, the change here will be reverted in the next pr.

File: src/components/HomepageFeatures/index.tsx (hunk 1 above)
Comment (SongYuQiu) [reply src/components/HomepageFeatures/index.tsx:4]: I missed this one last time, how about changing it to factory.png?

Comment (SongYuQiu) [reply src/components/HomepageFeatures/index.tsx:4]: This picture is temporarily used to occupy the place, and it will be deleted after the editor is implemented.

Comment (PragmaTwice) [reply src/components/HomepageFeatures/index.tsx:4]: Got it.

File: src/components/HomepageFeatures/index.tsx [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
-import feature0 from '/img/feature-0.png';
-import feature1 from '/img/feature-1.png';
-import feature2 from '/img/feature-2.png';
-import feature3 from '/img/feature-3.png';
-import feature4 from '/img/feature-4.png';
+import feature0 from '../../../static/img/feature-0.png';
+import namespace from '../../../static/img/namespace.png';
+import replication from '../../../static/img/replication.png';
+import highAvailable from '../../../static/img/high-available.png';
+import cluster from '../../../static/img/cluster.png'
```
[the remark this answers is not in the document]
Comment (SongYuQiu) [reply src/components/HomepageFeatures/index.tsx:8]: already done.

File: src/components/HomepageFeatures/index.tsx (hunk 2 above)
Comment (SongYuQiu) [reply src/components/HomepageFeatures/index.tsx]: reverted.

File: src/components/HomepageFeatures/index.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (PragmaTwice) [reply src/components/HomepageFeatures/index.tsx:8]: Emmm.. Seems no. It looks you solve these conflicts in the web-ui, which is not the normal way. You need to force pull the remote branch and then remove these files.

[the remark this answers is not in the document]
Comment (SongYuQiu) [reply src/components/HomepageFeatures/index.tsx:8]: Sorry for my mistake. The four old image files should have been removed by now.

File: docusaurus.config.js [hunk 4]
```diff
@@ -104,7 +104,7 @@ const config = {
         ],
       },
       footer: {
-        style: 'dark',
```
Comment (tisonkun) [inline docusaurus.config.js:107]: Please revert the changes on the footer. This is intended at https://github.com/apache/incubator-kvrocks-website/pull/69. We should avoid being up and down.

Comment (SongYuQiu) [reply docusaurus.config.js]: Sorry, I didn't notice #69 before. The footer has been reverted now.

File: yarn.lock [hunk 5]
```diff
@@ -1,11099 +1,7773 @@
...
-"locate-path@npm:^5.0.0":
-  version: 5.0.0
-  resolution: "locate-path@npm:5.0.0"
-  dependencies:
-    p-locate: ^4.1.0
-  checksum: 83e51725e67517287d73e1ded92b28602e3ae5580b301fe54bfb76c0c723e3f285b19252e375712316774cf52006cb236aed5704692c32db0d5d089b69696e30
-  languageName: node
-  linkType: hard
-
-"locate-path@npm:^6.0.
```
Comment (PragmaTwice) [inline yarn.lock:1]: The lockfile is completely re-generated. It looks unexpected.

Comment (SongYuQiu) [reply yarn.lock]: Sorry, I just installed anaconda and forgot to deactivate base. I simply re-executed the yarn command resulting in re-generating yarn.lock. I have rolled back the lockfile.

File: src/pages/index.module.css [hunk 6]
```diff
@@ -37,10 +59,35 @@
...
+    width:12rem;
+    margin-top: 1rem;
+  }
 }
+
+@media screen and (max-width: 996px) {
+  .heroBanner {
+    padding: 2rem;
+  }
+}
```
Comment (PragmaTwice) [inline src/pages/index.module.css:93]:
```suggestion
}

```

**Review by PragmaTwice — APPROVED** [review]
LGTM. Thanks for your contribution!
